### PR TITLE
[pwnn-1737] feat(cms): Refresh Case Statistics

### PR DESCRIPTION
**Pull request by cocoaken**

### CODE REVIEW

File: app/helpers/support/case_statistics_helper.rb [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
+    end
+
     def filter_cases_params(attributes = {})
       { filter_cases: attributes }
     end
+
+    def show_all_case_workers_link
+      unless session[:stats_selected_caseworker].nil?
+        content_tag(:span,
+                    link_to(I18n.t("support.case_statistics.selected_caseworker.show_all"),
```
Comment (EdwinKruglov) [inline app/helpers/support/case_statistics_helper.rb:32]: You can call `govuk_link_to` here (and in the views as well) with `no_visited_state: true` and remove the `govuk-link--no-visited-state` CSS class.

File: app/models/support/agent.rb [hunk 2]
```diff
@@ -10,12 +10,18 @@ class Agent < ApplicationRecord
     include RoleAssignable
 
     has_many :cases, class_name: "Support::Case"
+    has_many :live_cases, -> { live }, class_name: "Support::Case"
     belongs_to :support_tower, class_name: "Support::Tower", optional: true
     belongs_to :user, foreign_key: "dsi_uid", primary_key: "dfe_sign_in_uid", optional: true
 
+    scope :caseworkers, -> { by_role(%w[procops procops_admin]) }
+    scope :non_caseworkers, -> { caseworkers.invert_where }
```
Comment (EdwinKruglov) [inline app/models/support/agent.rb:18]: Didn't know about `invert_where`! Handy!

File: app/views/support/management/agents/index.html.erb
Comment (EdwinKruglov) [inline app/views/support/management/agents/index.html.erb:1]: Did you mean to include these changes in this PR?

**Review by EdwinKruglov — APPROVED** [review]

File: app/views/support/case_statistics/_by_category.html.erb [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  <div class="govuk-summary-card__title-wrapper">
+		<h2 class="govuk-summary-card__title">
+      <div class="govuk-summary-card__title_cell"><%= I18n.t("support.case_statistics.section_headings.by_category") %></div>
+      <div class="govuk-summary-card__title_cell"><%= show_all_case_workers_link %></div>
+    </h2>
+  </div>
+
+  <div class="govuk-summary-card__content">
+	  <div class="case-panel">
+			<table class="govuk-table overview-table overview-by-category">
```
Comment (EdwinKruglov) [inline app/views/support/case_statistics/_by_category.html.erb:11]: SonarCloud is complaining about these tables not having a description. You can probably add an `aria-describedby` and point to the `div` with the label or something similar.

File: app/views/support/management/agents/index.html.erb
Comment (cocoaken) [reply app/views/support/management/agents/index.html.erb:1]: Hey Ed, yes the thinking was to align the filter panel's "Former staff" language with the agent management's "Disabled accounts" (i.e. move everything to Current/Former staff, not Enabled/Disabled...)

File: app/helpers/support/case_statistics_helper.rb (hunk 1 above)
Comment (cocoaken) [reply app/helpers/support/case_statistics_helper.rb]: Blimey, you live and learn. I've done that in all the views now, thanks!

**Pull request merged**